pyvrp/tests/helpers.py

```diff
@@ -37,6 +38,13 @@ def sleep(duration, get_now=time.perf_counter):
         now = get_now()
 
 
+def make_random_solutions(num_sols, pm, rng, data):
+    """
+    Returns a list of a ``num_sols`` random solutions.
+    """
+    return [Individual.make_random(data, pm, rng) for _ in range(num_sols)]
```

Comment:
Can we have the order of the arguments here be the same as they are for almost every other thing: `data, pm, rng`? I'm fine with `num_sols` coming first.